### PR TITLE
[#955] Jupyter inside helm

### LINKED ISSUE #955: Add JupyterLab instance inside Legion's HELM chart
As a user of platform I want to have an ability to use JupyterLab after deploying of Legion Platform on top of Kubernetes.

### PR DESCRIPTION
**Pull request by kirillmakhonin**
This closes #955

### CODE REVIEW

**Comment by kirillmakhonin** [comment]
bot, test all

File: helms/legion/values.yaml [hunk 1]
```diff
@@ -573,6 +573,62 @@ edi:
...
+  # Type: string
+  # image: custom-image:1.0
+
+  # Port on which JupyterLab listens income traffic
+  # Type: integer
+  port: 80
+
+  # Resources for each instance
+  # For declaration format see https://kubernetes.io/docs/concepts/configuration/manage-compute-resources-container/
+  resources:
```
Comment (mcdoker18) [inline helms/legion/values.yaml:594]: IMHO, we should increase the default resources.

File: helms/legion/values.yaml [hunk 2]
```diff
@@ -573,6 +573,62 @@ edi:
...
+    requests:
+      cpu: "50m"
+      memory: "128Mi"
+    limits:
+      cpu: "100m"
+      memory: "512Mi"
```
Comment (mcdoker18) [inline helms/legion/values.yaml:597]: Please, add a robot test. It should check that the jupiterlab deployment is up and closed by dex.

**Review by mcdoker18 — CHANGES_REQUESTED** [review]

**Review by karbyshevds — DISMISSED** [review]

File: helms/legion/values.yaml (hunk 1 above)
Comment (mcdoker18) [reply helms/legion/values.yaml:594]: Fixed

File: helms/legion/values.yaml (hunk 2 above)
Comment (mcdoker18) [reply helms/legion/values.yaml:597]: Fixed

**Comment by mcdoker18** [comment]
bot, test app

**Comment by mcdoker18** [comment]
bot, test gcp app

**Review by alexander-semenets — APPROVED** [review]

**Review by karbyshevds — APPROVED** [review]

**Review by maxkochubey — APPROVED** [review]

**Review by mcdoker18 — APPROVED** [review]

**Pull request merged**